src/lilgit/model/sampler.ts

```diff
@@ -253,100 +253,158 @@ export class Sampler {
     return JSON.stringify(nodey.raw);
   }
 
-  public async renderDiff(
+  public async renderArtifactCell(
     nodey: Nodey,
     elem: HTMLElement,
-    options: {
-      newText?: string;
-      diffKind?: number;
-      textFocus?: string;
-    } = {}
+    type: CELL_TYPE,
```

Comment:
What is the goal of the CELL_TYPE enum? Would it be possible to do the switch statement instead on the nodey.typeChar property?